config/config.go

```diff
@@ -246,6 +253,7 @@ type (
 		CandidateIndexDBPath   string           `yaml:"candidateIndexDBPath"`
 		StakingIndexDBPath     string           `yaml:"stakingIndexDBPath"`
 		ID                     uint32           `yaml:"id"`
+		ExternChainID          uint32           `yaml:"externChainID"`
```

Comment:
we may have a more precise name which can better explain itself from its name

Reply:
so do you have a suggestion?

Reply:
https://chainid.network/ EVM_NETWORK_ID

Reply:
i don't want to explicitly call out the name EVM, also this extern chain ID can in theory be used for another type of wire encoding format -- though that's not very likely in the near future

Reply:
it should be explicit and shouldn't be shared by other encoding type